src/firejail/x11.c

```diff
@@ -625,6 +625,52 @@ void x11_start_xephyr(int argc, char **argv) {
 }
 
 
+// this function returns the string that will appear in the window title when xpra is being used
+// this string may include one of these items:
+// *  the "--name" argument, if specified
+// *  the basename portion of the "--private" directory, if specified
+// note: the malloc() is leaking, but this is a small string allocated one time only during startup, so don't care
+static char * get_title_arg_str() {
+
+	char * title_arg_str = NULL;
+
+	const char * title_start = "--title=firejail x11 sandbox";
+	const char * title_sep = " ";
+
+	// use the "--name" argument if it was explicitly specified
+	if ((cfg.name != NULL) && (strlen(cfg.name) > 0)) {
+
+		title_arg_str = malloc(strlen(title_start) + strlen(title_sep) + strlen(cfg.name) + 1);
```

Comment:
Could you check the return values of the malloc calls?

Reply:
Done.  Commit 7f39af86.